catalog/lib/widgets/app_dropdown.dart

```diff
@@ -0,0 +1,82 @@
+import 'package:catalog/catalog.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+
+class AppDropdownMenuEntry<T> {
+  final T value;
+  final String label;
+
+  const AppDropdownMenuEntry({
+    required this.value,
+    required this.label,
+  });
+}
+
+class AppDropdownMenu<T> extends StatefulWidget {
+  final List<AppDropdownMenuEntry<T>> dropdownMenuEntries;
```

Comment:
Why not take a list of items directly, instead of list of AppDropdownMenuEntry<T> ?
It ends up being the same as using the menu button directly but without the style, and the style we can change it in theme